pkgs/development/libraries/cmakerc/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
```

Comment:
Does this software need GCC?

Reply:
I don't think so

Reply:
Then use `stdenvNoCC` instead of `stdenv`.